modules/openapi-generator-cli/pom.xml

```diff
@@ -4,7 +4,7 @@
         <groupId>org.openapitools</groupId>
         <artifactId>openapi-generator-project</artifactId>
         <!-- RELEASE_VERSION -->
-        <version>5.0.1-SNAPSHOT</version>
+        <version>5.1.0-SNAPSHOT</version>
```

Comment:
I think version bump should go in a separate PR, but leaving this to @wing328

Reply:
Wrong target branch